### PR TITLE
fix the makefile file and fix the cpu occupancy problem in Win10 when using 'gnuplot -persist'

### PR DESCRIPTION
**Pull request by zoziha**
# 1. `gnuplot -persist` problem in Win10
I have pushed a issue (#15 ) about `gnuplot -persist` problem in Win10.
When running `gnuplot --persist *.gp`, the cpu is fully loaded (100%),
I think the `gnuplot --persist` command caused this problem.
image

**The CPU occupancy is high**, which may be a **Windows-gnuplot's bug**. After testing, it is found that using the `wgnuplot -persist` command under Windows will not cause unexplained CPU occupancy problems.
image

# 2. update makefile
I found that the folder structure of "ogpf" has changed, and the makefile is also invalid. I easily **updated** the makefile to make it available and valid.

### CODE REVIEW

File: src/ogpf.f90 [hunk 1]
```diff
@@ -2248,8 +2248,34 @@ subroutine finalize_plot(this)
         this%hasfileopen = .false.        ! reset file open flag
         this%hasanimation = .false.
         ! Use shell command to run gnuplot
-        call execute_command_line ('gnuplot -persist ' // this%txtfilename)  !   Now plot the results
-
+        if (get_os_type() == 1) then
+            call execute_command_line ('wgnuplot -persist ' // this%txtfilename)  !   Now plot the results
```
Comment (Brookszzh) [inline src/ogpf.f90:2252]: I set up a judgment (whether the current system is the windows system) to choose which command line to run.

Comment (kookma) [reply src/ogpf.f90:2252]: Thank you! Sounds good!

**Comment by kookma** [comment]
@LKedward would you please have a look!

@zoziha Thank you

**Comment by LKedward** [comment]
Thanks @zoziha, these changes look good to me :+1:

File: src/ogpf.f90 (hunk 1 above)
Comment (zoziha) [reply src/ogpf.f90:2252]: It seems that the `set term wxt` command in the script file "*.gp" caused the cpu to be occupied when running `gnuplot -persist` under the Windows system.

Thank you for your adoption! 😋@kookma @LKedward 

set term wxt

**Pull request merged**